tests/hooks/test_subprocess.py

```diff
@@ -96,3 +96,9 @@ def test_should_exec_subprocess(self, mock_popen, mock_temporary_directory):
             stderr=STDOUT,
             stdout=PIPE,
         )
+
+    def test_task_decode(self):
+        hook = SubprocessHook()
+        command=['bash', '-c', "source "+__file__[:-2]+'sh']
+        result = hook.run_command(command=command)
+        assert result.exit_code==0
```

Comment:
Instead of needing a separate script, I think you can just use `sys.executable` to print some weird characters.

Reply:
Simplified the test, so the script isn't needed anymore